alternative-install/docker-compose/images/backend-module/docker-entrypoint

```diff
@@ -0,0 +1,175 @@
+#!/usr/bin/env /bin/bash
+
+set -o pipefail -o noclobber
+
+# keep for backward compatibility
+if [ "$MODULE_URL" != "" ]; then
+  SERVICE_URL=$MODULE_URL
+fi
+
+module_descriptor=''
+
+# wait for okapi coming up and ready
+wait_for_okapi() {
+  echo -ne "waiting for okapi at ${OKAPI_URL} to become ready ..."
+  while [ true ]; do
```

Comment:
Can you improve the readability of the loop?

Example (POSIX-compliant, works in dash):

    for i in `seq $OKAPI_TRY_COUNT`; do